iup/models.py

```diff
@@ -610,3 +611,38 @@ def date_to_interval(date_col: pl.Expr) -> pl.Expr:
         Time difference is always in days.
         """
         return date_col.diff().dt.total_days().cast(pl.Float64)
+
+    @staticmethod
+    def insert_rollout(
```

Comment:
Now `insert_rollout` is in the right place, but it still isn't being deployed. Fixing that seems out-of-scope for this PR though. I filed this into issue #77 and will deal with it later this sprint.